pyproject.toml

```
@@ -33,7 +33,7 @@ all = [
     "pydocstyle>=2.0.0",
     "pyflakes>=2.4.0,<2.5.0",
     "pylint>=2.5.0",
-    "rope>=0.10.5",
+    "rope>=1.1.1",
```

```suggestion
    "rope>1.2.0",
```

To match with the requirement below.

---

Fixed in a8814981c6447dc6b352bbbb05db18c3df82aeb0